gobblin-docs/user-guide/Docker-Integration.md

```diff
@@ -18,68 +18,68 @@ The `gobblin/gobblin-wikipedia` repository contains images that run the Gobblin
 
-The Docker images for this repository can be found on Docker Hub [here](https://hub.docker.com/r/gobblin/gobblin-wikipedia/). These images are mainly meant to act as a "Hello World" example for the Gobblin-Docker integration, and to provide a sanity check to see if the Gobblin-Docker integration is working on a given machine. The image contains the Gobblin configuration files to run the [Gobblin Wikipedia job](../Getting-Started). When a container is launched using the `gobblin-wikipedia` image, Gobblin starts up, runs the Wikipedia example, and then exits.
+### Set working directory
 
-Running the `gobblin-wikipedia` image requires taking following steps (lets assume we want to an Ubuntu based image):
+Before running docker containers, set a working directory for Gobblin jobs:
 
-* Download the images from the `gobblin/gobblin-wikipedia` repository
+`export LOCAL_JOB_DIR=<local_gobblin_directory>`
 
-```
-docker pull gobblin/gobblin-wikipedia:ubuntu-gobblin-latest
-```
+We will use this directory as the [volume](https://docs.docker.com/storage/volumes/) for Gobblin jobs and outputs. Make sure your Docker has the [access](https://docs.docker.com/docker-for-mac/#file-sharing) to this folder. This is the prerequisite for all following example jobs.
 
-* Run the `gobblin/gobblin-wikipedia:ubuntu-gobblin-latest` image in a Docker container
+### Run the docker image with simple wikipedia jobs
 
-```
-docker run gobblin/gobblin-wikipedia:ubuntu-gobblin-latest
-```
+Run these commands to start the docker image:
 
-The logs are printed to the console, and no errors should pop up. This should provide a nice sanity check to ensure that everything is working as expected. The output of the job will be written to a directory inside the container. When the container exits that data will be lost. In order to preserve the output of the job, continue to the next step.
+`docker pull gobblin/gobblin-standalone:alpine-gaas-latest`
 
-* Preserving the output of a Docker container requires using a [data volume](https://docs.docker.com/engine/tutorials/dockervolumes/). To do this, run the below command:
+`docker run gobblin/gobblin-standalone:alpine-gaas-latest`
```

Comment:
This wouldn't pick up any jobs unless we expose `LOCAL_JOB_DIR` as a docker volume to `/tmp/gobblin-standalone/jobs`, where gobblin standalone picks up its jobs. Usually not a problem in docker-compose where we can define volumes but if we want to run the docker image directly it'd look something like this 

`docker run -v $LOCAL_JOB_DIR:/tmp/gobblin-standalone/jobs gobblin/gobblin-standalone:latest`

Also when I tried running the wikipedia example it gives me this error: 
```
Thread Thread[JobScheduler-0,5,main] threw an uncaught exception: java.lang.VerifyError: Stack map does not match the one at exception handler 77
```

Did we confirm that the example is still working?

Reply:
Agree for the docker volume setup and changed as you proposed.

The uncaught exception was newly introduced by some recent PR. It's not only happening in docker run, but also in direct run from local build. I have raised the discussion in slack. Looks like it was introduced by dependency issue with Jackson.

Reply:
The wiki pull sample job works fine.